docs/source/cuda/cooperative_groups.rst

```diff
@@ -0,0 +1,120 @@
+==================
+Cooperative Groups
```

Comment:
Reminder from OOB chat to consider adding a note about Anaconda CTK version 10+ needed unless backport of static libs is done.

Reply:
Added.